### PR TITLE
Enhanced Campaign detail page with various data and features

### PR DESCRIPTION
**Pull request by rpnunez**

### CODE REVIEW

File: ai-post-scheduler/assets/js/campaigns.js [hunk 1]
```diff
@@ -182,6 +183,56 @@
...
+
+			$button.prop('disabled', true).addClass('is-busy');
+
+			$.ajax({
+				url: ajaxurl,
+				type: 'POST',
+				data: {
+					action: 'aips_run_now',
+					nonce: aipsAjax.nonce,
+					schedule_id: scheduleId
```
[the remark this answers is not in the document]
Comment (rpnunez) [reply ai-post-scheduler/assets/js/campaigns.js:218]: This action is using the Schedule controller's 'aips_run_now' method. No need to register a new method that does the same thing.
